### ISSUE TITLE
Incorrect DESTINATION_REPOSITORY value with release v2.12

### ISSUE THREAD

**Issue by dcrossleyau** [repo-sync/pull-request#126]
This pull-request action has been working beautifully for us for a long time. Thanks so much.

However today it broke with the v2.12 release.

Our repo is public:
https://github.com/folio-org/folio-org.github.io/actions/workflows/pr-config-apidocs.yml

For today's failed runs, see the "Gather Inputs" section where it has the incorrect value:\
`DESTINATION_REPOSITORY=folio-org/folio-org`\
i.e. the organization name instead of the repository name.

See the previous successful run which has the correct value:\
`DESTINATION_REPOSITORY=folio-org/folio-org.github.io`

**Comment by wei** [comment]
Thanks for the report! For now please use v2.11. We'll fix this soon!

@m4heshd any way you can revise the repo matching to support all valid characters including ._- symbols, etc?

**Comment by m4heshd** [comment]
@wei I'll take a look at it just in a few hours. I was waiting for something to go south after the release. 😆

**Comment by m4heshd** [comment]
@dcrossleyau Can you do a test using:

```yaml
- name: pull-request
  id: open-pr
  uses: m4heshd/pull-request@special-char-fix
```

**Comment by dcrossleyau** [comment]
Hooray, yes that is verified. @m4heshd -- Thanks for your prompt attention.

See [successful run](https://github.com/folio-org/folio-org.github.io/actions/runs/3972665667).

**Comment by wei** [comment]
This is now released as repo-sync/pull-request@v2.12.1 and repo-sync/pull-request@v2 🎉

**Comment by dcrossleyau** [comment]
Thanks to the responsiveness of all.
